Detectors/TOF/workflow/src/TOFRawWriterSpec.cxx

```diff
@@ -86,6 +100,19 @@ void RawWriter::run(ProcessingContext& pc)
   encoder.flush();
   encoder.close();
 
+  // create configuration file for rawreader
+  std::ofstream fconfig;
+  fconfig.open("TOFraw.cfg");
+  fconfig << "[defaults]\n";
+  fconfig << "dataOrigin = TOF\n";
+  fconfig << "dataDescription = RAWDATA\n\n";
+
+  for (int icru = 0; icru < encoder.getNCRU(); icru++) {
+    fconfig << "[input-" << icru << "]\n";
```

Comment:
To avoid ``input-xxx`` being identical in config files obtained by concatenation of those from different detectors, could you register it as ``[input-tof-" << icru << "]``? The parser looks only for unique ``input<something>``...

Reply:
Done